CHANGELOG.md

```diff
@@ -1,8 +1,87 @@
 
+## [1.41](https://github.com/brave/brave-ios/releases/tag/v1.41)
+
+- Updated Ecosia search engine logo under search engine settings. ([#5504](https://github.com/brave/brave-ios/issues/5504))
+- Updated URL bar and bottom toolbar transition during page scroll. ([#5645](https://github.com/brave/brave-ios/issues/5645))
+- Fixed 'Try Brave Search' button to show full text on multi-line for certain languages. ([#5644](https://github.com/brave/brave-ios/issues/5644))
+- Fixed Wallet panel from being shown during page reload. ([#5616](https://github.com/brave/brave-ios/issues/5616))
+
+## [1.40](https://github.com/brave/brave-ios/releases/tag/v1.40.1)
+
+- Implemented Get Encryption Public Key & Decrypt Requests for wallets. ([#5424](https://github.com/brave/brave-ios/issues/5424))
+- Added context menu to show full account address. ([#5477](https://github.com/brave/brave-ios/issues/5477))
+- Added badge to wallet URL bar icon when there is a pending request. ([#5358](https://github.com/brave/brave-ios/issues/5358))
+- Fixed tab from loading YouTube home page when browser is restarted. ([#5615](https://github.com/brave/brave-ios/issues/5615))
+- Fixed Reader mode to use custom fonts. ([#5553](https://github.com/brave/brave-ios/issues/5553))
+- Fixed to show eTLD+1 prominently in request panel ([#5188](https://github.com/brave/brave-ios/issues/5188))
+- Fixed wallet to show eTLD+1 in bold and show the full URL. ([#5534](https://github.com/brave/brave-ios/issues/5534))
+- Fixed wallet to show eTLD+1 in bold on transaction confirmation screens. ([#5405](https://github.com/brave/brave-ios/issues/5405))
+- Fixed wallet provider object to be only available for `https` sites. ([#5518](https://github.com/brave/brave-ios/issues/5518))
+- Fixed wallet blockie colors to match other platforms. ([#5472](https://github.com/brave/brave-ios/issues/5472))
+- Fixed white screen when restoring a tab via tab tray. ([#5508](https://github.com/brave/brave-ios/issues/5508))
+
+## [1.39.1](https://github.com/brave/brave-ios/releases/tag/v1.39.1)
+
+- Fixed page from going into infinite redirect loop when trying to load an amp page. ([#5461](https://github.com/brave/brave-ios/issues/5461))
+
+## [1.39](https://github.com/brave/brave-ios/releases/tag/v1.39)
+
+- Implemented debouncing feature on iOS. ([#5225](https://github.com/brave/brave-ios/issues/5225))
+- Implemented additional farbling protections for fingerprinters accessing CPU information. ([#5268](https://github.com/brave/brave-ios/issues/5268))
+- Added "Force Paste" option to long press context menu. ([#5292](https://github.com/brave/brave-ios/issues/5292))
+- Added ability to set default currency via wallet settings. ([#4341](https://github.com/brave/brave-ios/issues/4341))
+- Fixed Share menu from showing up when selecting "Share Link" via Mobile bookmarks. ([#5331](https://github.com/brave/brave-ios/issues/5331))
```

Comment:
Probably a good idea to add the quotes around `Share` & `Mobile` to make it easier to read. Example:

* Fixed "Share" menu from showing up when selecting "Share Link" via "Mobile" bookmarks.